Find a functionally equivalent code:android.widget.ListPopupWindow.setModal:COMMENT
Method Modifier: public      
Comment:/**
 * Set whether this window should be modal when shown.
 *
 * <p>If a popup window is modal, it will receive all touch and key input.
 * If the user touches outside the popup window's content area the popup window
 * will be dismissed.
 *
 * @param modal {@code true} if the popup window should be modal, {@code false} otherwise.
 */

@@ -1,4 +1,4 @@
 {
-    mModal = true;
+    mModal = modal;
     mPopup.setFocusable(modal);
 }

